mne/io/eeglab/eeglab.py

```diff
@@ -525,8 +525,13 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
     if event_id is None:
         event_id = dict()
 
-    types = [event.type for event in eeg.event]
-    latencies = [event.latency for event in eeg.event]
+    try:
+        types = [event.type for event in eeg.event]
+        latencies = [event.latency for event in eeg.event]
+    except TypeError:
```

Comment:
Yes, that's what I was proposing with `isinstance` - it works well, I've tested it with 0, 1 and more events :) Ok, I'll push the change.